navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/milestone/TriggerProperty.java

```diff
@@ -55,4 +61,35 @@
   public static final int TRUE = 0x00000124;
 
   public static final int FALSE = 0x00000100;
+
+  static SparseArray<Number[]> getSparseArray(RouteProgress previousRouteProgress, RouteProgress routeProgress) {
+    // Build hashMap matching the trigger properties to their corresponding current values.
+    SparseArray<Number[]> statementObjects = new SparseArray<>();
```

Comment:
Should we set the `SparseArray` initial capacity to `10` for each statement object?